pkg/server/service_test.go

```diff
@@ -6,25 +6,34 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func mockServiceInMemory() Service {
+func mockServiceInMock() Service {
+	repository := NewRepositoryInMemory(nil)
+	return NewMockService(repository)
+}
+func mockServiceInReal() Service {
 	repository := NewRepositoryInMemory(nil)
 	return NewService(repository)
 }
+
+const Hashicorp_cloud_key = "hashicorp cloud key"
+const Local_vault_key = "my-fixed-token"
+const Hashicorp_cloud_url = "https://vault-cluster-public-vault-2d92a425.16ce2ded.z1.hashicorp.cloud:8200"
```

Comment:
Do not hardcode credentials! You can read environment variables with `os.Getenv` at runtime.